pkg/distro/image_config.go

```diff
@@ -94,6 +94,9 @@ type ImageConfig struct {
 	LockRootUser *bool
 
 	IgnitionPlatform *string
+
+	// How to handle the /etc/machine-id file
+	MachineIdUninitialized *bool
```

Comment:
It'd be useful to expand on this.  Since the option is a boolean "how to handle ..." doesn't say much.  Let's clarify what happens when it's enabled, like "When enabled, sets the machine ID to `uninitialized`, which causes first boot operations to run."
(or something like that)